src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/profiledetails/ProfileDetailsPhysicalAttributesMigrationIntTest.kt

```diff
@@ -70,9 +64,7 @@ class ProfileDetailsPhysicalAttributesMigrationIntTest : SqsIntegrationTestBase(
       @BeforeEach
       fun setUp() = runTest {
         stubMigrationDependencies(entities = 2)
-        // TODO SDIT-2019 replace with a call to the start migration endpoint when it's done
-        migrationId = migrationService.startMigration(PrisonPersonMigrationFilter(migrationType = PROFILE_DETAILS_PHYSICAL_ATTRIBUTES)).migrationId
-          .also { waitUntilCompleted() }
+        migrationId = webTestClient.performMigration().migrationId
```

Comment:
Now testing against the start migration endpoint instead of calling the service directly.